dist/dotty.rb

```diff
@@ -0,0 +1,32 @@
+class Dotty < Formula
+  desc "Experimental Scala Compiler"
+  homepage "http://dotty.epfl.ch/"
+  url "http://dotty.epfl.ch/dotty/dotty-0.1.1-bin-SNAPSHOT.tar.gz"
```

Comment:
Here is the script I used to upload artifacts to github releases and update a homebrew formula https://github.com/scalameta/scalafmt/blob/ad8559939c7d32ca78dfedf3ebc4eebc2fda7257/bin/publish.sh#L52-L71 I don't use it anymore, I only publish to Maven Central now